flowermd/base/system.py

```diff
@@ -610,7 +610,7 @@ def visualize(self):
             )
 
 
-class Pack(System):
+class Pack(System, UniqueMolecules=True):
```

Comment:
We should move this parameter down to the `__init__` method, and for consistency with other parameter names, we should use snake case `unique_molecules`.